src/scss/grommet-core/_objects.card.scss

```diff
@@ -39,7 +39,14 @@ $card-hover-color: #EBEBEB;
     }
   }
 
-  // &__thumbnail {
+  &.#{$grommet-namespace}box--responsive {
```

Comment:
.grommetux-box--width-medium, and margins persist in mobile:
![mobile-margin](https://cloud.githubusercontent.com/assets/10161095/18463808/596a2fb0-7928-11e6-8588-31e60f0da799.png)

This section overrides the width and margin:
![mobile-margin-fix](https://cloud.githubusercontent.com/assets/10161095/18461368/858afa5c-7914-11e6-87f6-47d9fb7f1aea.png)